File: SoftInvModel/src/main/java/pe/edu/pucp/softinv/model/Personas/EmpleadoDTO.java
package pe.edu.pucp.softinv.model.Personas;

import pe.edu.pucp.softinv.model.Servicio.ServicioDTO;

import java.util.ArrayList;

public class EmpleadoDTO extends UsuarioDTO{
    private Boolean admin;
    private ArrayList<ServicioDTO> servicios;

    @Override
    public void setRol(){
        if(admin) rol="Admin";
        else rol="Empleado";
    }

    public void setRol(String rol){
        this.rol=rol;
    }

    public ArrayList<ServicioDTO> getServicios() {
        return servicios;
    }

    public void setServicios(ArrayList<ServicioDTO> servicios) {
        this.servicios = servicios;
    }

    public EmpleadoDTO() {
        super();
        admin=null;
        servicios=null;
    }

    public EmpleadoDTO(String nombre, String PrimerApellido, String SegundoApellido, String correoElectronico,
                       String contrasenha, String celular,Integer idUsuario, Boolean admin,String urlFotoPerfil,ArrayList<ServicioDTO> servicios) {
        super(nombre, PrimerApellido, SegundoApellido, correoElectronico, contrasenha, celular, urlFotoPerfil,idUsuario);
        this.admin=admin;
        this.servicios=servicios;
        setRol();
    }

    public Boolean isAdmin() {
        return admin;
    }

    public void setAdmin(Boolean admin) {
        this.admin = admin;
    }

}

File: C#/SoftInvModel/Personas/EmpleadoDTO.cs
using System;
using System.ComponentModel;

namespace softinv.model
{
    public class EmpleadoDTO : UsuarioDTO
    {
        private bool? admin;
        private BindingList<ServicioDTO> servicios;

        public bool? Admin { get => admin; set => admin = value; }
        internal BindingList<ServicioDTO> Servicios { get => servicios; set => servicios = value; }

        public override void setRol()
        {
            if ((bool)Admin) rol = "Admin";
            else rol = "Empleado";
        }

        public void setRol(string rol)
        {
            this.rol = rol;
        }


        public EmpleadoDTO() : base()
        {
            Admin = null;
            Servicios = null;
        }

        public EmpleadoDTO(string nombre, string PrimerApellido, string SegundoApellido, string correoElectronico,
                           string contrasenha, string celular, int idUsuario, bool admin, string urlFotoPerfil, BindingList<ServicioDTO> servicios) : base(nombre, PrimerApellido, SegundoApellido, correoElectronico, contrasenha, celular, urlFotoPerfil, idUsuario)
        {
            this.Admin = admin;
            this.Servicios = servicios;
            setRol();
        }

        public EmpleadoDTO(bool admin, BindingList<ServicioDTO> servicios) : base()
        {
            this.Admin = admin;
            this.Servicios = servicios;
        }
    }
}